docs/specs/schema.yml

```diff
@@ -136,7 +136,7 @@ outputs:
   toc.json:
   docs/toc.json:
   .errors.log: |
-    ["warning","unexpected-type","Expect type 'String' but got 'Object'","docs/a.yml",3,18]
+    ["suggestion","unexpected-type","Expected type 'String' but got 'Object'","docs/a.yml",3,18]
```

Comment:
This error code is shared with other scenarios, changing it to suggestion breaks existing build error level.

Reply:
sure, i will change back